pkgs/development/compilers/unison/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, gnutar }:
+
+stdenv.mkDerivation rec {
+  name = "unison-code-manager-${version}";
+  milestone_id = "M1c";
+  version = "0.0.1-alpha-${milestone_id}";
+
+  src = if (stdenv.isDarwin) then
+    fetchurl {
+      url = "https://github.com/unisonweb/unison/releases/download/release/${milestone_id}/unison-osx.tar.gz";
+      sha256 = "03q02r7qc7ybqz16kmpk2d8l9vx28kaj9x59mlxzi8a4mr0j3vzb";
+    }
+  else
+    fetchurl {
+      url = "https://github.com/unisonweb/unison/releases/download/release/${milestone_id}/unison-linux64.tar.gz";
+      sha256 = "1iwynqnp1i39pyq9wc01x7y22y1qa0rrjlx40jjdgnj23y1r6jk4";
+    };
+
+  phases = "installPhase";
+
+  # There must be a simpler "just unpack and copy, dammit" standard derivation.
+  installPhase = ''
+    mkdir -p ${placeholder "out"}/bin
```

```suggestion
    mkdir -p $out/bin
```

I don't see this done typically in `installPhase`